src/areas/areas.service.ts

```diff
@@ -0,0 +1,242 @@
+import { Area } from '@prisma/client';
+import { PrismaService } from 'src/prisma.service';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { CreateAreaDto } from 'src/models/area/create-area.dto';
+import { UpdateAreaDto } from 'src/models/area/update-area.dto';
+
+@Injectable()
+export class AreasService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  /**
+   * Creates a new area.
+   * @param createAreaDto - Data Transfer Object for creating a new area.
+   * @returns A promise that resolves with a message, any error encountered, and the created area data.
+   */
+  async create(
+    createAreaDto: CreateAreaDto,
+  ): Promise<{ message: string; error: string | null; data: Area | null }> {
+    try {
+      const area = await this.prisma.area.create({
+        data: {
+          ...createAreaDto,
+        },
```

Comment:
Avoid to use spread operator in parameter as data
```ts
data: createAreaDto
```
instead of
```ts
data: { ...createAreaDto } // ❌ wrong implementation, bad for type safety
```